Battery Pack V1 Modification

SWTORboi opened the pull request:
Modified the battery pack traces by flipping footprint to replace part +/- and replacing footprints of oversized holders

SWTORboi committed:
branch_upload_test

SWTORboi committed:
Modified traces to be out of way of mounting holes

Mikefly123 commented:
Hey @SWTORboi! Sorry it took so long to circle back to this, I just looked at your edits and overall they look great! 

One major error on the board to clean up:
<img width="1144" alt="Screenshot 2025-05-13 at 7 27 27 PM" src="https://github.com/user-attachments/assets/3c0831b2-83ea-4d00-9fd4-486005ec7997" />
Was the connector intentionally switched to a DF11 through hole? The one on the original V1 battery pack was a surface mounted DF11 connector.

Two housekeeping issues:
1. It looks like the way that the schematic was uploaded caused it to break. This error appears when I try to open it:
<img width="314" alt="Screenshot 2025-05-13 at 7 30 06 PM" src="https://github.com/user-attachments/assets/df7d9f49-ba33-46ff-b9c3-64d952466d63" />

2. It would be good if rather than just overwriting the V1 battery pack project we generate a V1a battery pack project. You can do this easily by just going to file > save as

SWTORboi committed:
Mod

SWTORboi committed:
rename

SWTORboi committed:
Fixed DF11 footprint and added new (working) repo

Working repo is pack_v1a (ignore pack_v1, will restore original version later in this branch)

SWTORboi commented:
Hi @Mikefly123 , fixed the footprint issue and made the new v1a project folder on this branch, the v1 .sch still is crashing so we can either ignore it and just merge the v1a with main eventually or I can fix it later

Mikefly123 committed:
Addressed DRC Errors

Mikefly123 committed:
Nudged Around Some Trace

Mikefly123 committed:
Pulled back two more connections on the DF11

Mikefly123 committed:
Generated Production Files

Mikefly123 commented:
## V1a Schematic 
![image](https://github.com/user-attachments/assets/e1dc6bbb-d6cb-4f9a-800a-845cd79fd950)

## V1a Layout
Top
![image](https://github.com/user-attachments/assets/ae658281-fa0c-49cd-9910-a0bb885d41f6)
Bottom
![image](https://github.com/user-attachments/assets/efa7d497-d270-4543-8f68-42bf5bc680dc)

Mikefly123 reviewed (approved):
Hey @SWTORboi! The changes look good to me. I did a little cleanup, and now this is good to merge.
